app/controllers/coin_articles_controller.rb

```diff
@@ -1,14 +1,15 @@
 class CoinArticlesController < ApplicationController
   rescue_from ActiveRecord::RecordNotFound, :with => :record_not_found
   before_action :set_coin_article, only: [:show]
+  prepend_before_action :set_amp, only: [:index]
 
   breadcrumb 'How to Buy Cryptocurrency', :coin_articles_path, match: :exact
 
   def index
     distribute_reads(max_lag: MAX_ACCEPTABLE_REPLICATION_LAG, lag_failover: true) do
       @coin_articles = CoinArticle.page(params[:page]).per(params[:limit])
+      @coin_articles.each # force load
```

Comment:
Thanks for the comment! Just curious why force loading is required here?

Reply:
distribute_reads needs to force load to work properly. normally the recommended method is `to_a`, but that would get rid of the methods provided by Kaminari
distribute_reads also suggests `load`, but that didn't seem to be working either